Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/Util/DatabaseHelper.swift

```diff
@@ -717,11 +717,29 @@ class DatabaseHelper {
         // Those migrations were some heavy DROP COLUMN that we moved outside of DB startup
         if schemaVersion < 48 {
             schemaVersion = 48
+            do {
+                try db.executeUpdate("ALTER TABLE SJEpisode DROP COLUMN contentType", values: nil)
```

Comment:
I'm not sure if we should put this back here but my rational is:
- Some users maybe be on versions where where the  clean up process is not run, so they will have the new columns, so we need to drop it and then create it again on version 50.
- The reason for the above is that cleanUp code is run after this update code.

Reply:
I think what we might be able to do is to just remove the `DROP TABLE` from everywhere.

Then on `50` you can do:

```
try? db.executeUpdate("ALTER TABLE SJEpisode ADD COLUMN contentType TEXT;", values: nil)
try? db.executeUpdate("ALTER TABLE SJUserEpisode ADD COLUMN contentType TEXT;", values: nil)
```

If the column exists, this will fail but we're using `try?`. If not, it will be created. This way we avoid the unnecessary work of removing and re-adding for those users.

I have to test this though.

Reply:
@leandroalonso good suggestion, just changed the code to use try?.